index.js

```diff
@@ -28,11 +32,12 @@ module.exports = {
           throw new Error('@apostrophecms/passport-bridge: you must configure the "strategies" option');
         }
 
-        for (const spec of self.options.strategies) {
+        for (const strategy of self.options.strategies) {
+          const spec = structuredClone(strategy);
           // Works with npm modules that export the strategy directly, npm modules
           // that export a Strategy property, and directly passing in a strategy property
           // in the spec
-          const strategyModule = spec.module && await self.apos.root.import(spec.module);
+          const strategyModule = spec.module && await import(spec.module);
```

Comment:
I am not sure this will work when passport-bridge is npm linked, because the search path might not include the root of the project, but that's just an "us problem."

Reply:
And I'm OK with it, if it turns out to be true, as long as we document how to work around it and debug the module.

Reply:
I did not use `npm link` but `npm install ../passport-bridge` (with a relative path).

Without this change, when using a relative path, it tries to find the passport strategy in `node_modules/apostrophe/node_modules/`.

I'll do more testing today and on Monday.